src/shared/constants/DeveloperUtils.js

```diff
@@ -0,0 +1,81 @@
+import { compareSync } from 'bcrypt';
```

Comment:
it's not really recommended to move a file and then make changes within that moved file since it's difficult to spot changes you actually made (since the entire file is now green in GitHub) - ideally, if you want to refactor the codebase, you would just move the file.

please only move the file in this PR and then in future PRs you can suggest making changes

Reply:
Okay